packages/admin/stubs/TableWidget.stub

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace {{ namespace }};
+
+use Closure;
+use Filament\Tables;
+use Filament\Widgets\TableWidget as BaseWidget;
+use Illuminate\Database\Eloquent\Builder;
+
+class {{ class }} extends BaseWidget
+{
+    protected function getTableHeading(): string | Closure | null
+    {
+        return 'Table';
+    }
+
+    protected function getTableQuery(): Builder
+    {
+        return null;
```

Comment:
Perhaps the stub should be empty instead of having an invalid `return` type?

Reply:
I would leave it as an empty `// ...` line or something.

Reply:
Cool, done.